Sprint-1/exercises/paths.js

```diff
@@ -15,4 +15,6 @@ const base = filePath.slice(lastSlashIndex + 1);
 console.log(`The base part of ${filePath} is ${base}`);
 
 // Create a variable to store the dir part of the filePath variable
+const directory = "/Users/mitch/cyf/Module-JS1/week-1/interpret";
```

Comment:
You were suppose to use a function (similar to the one at line 14) to get the directory part from `filePath`.

Similarly, you need to use an approach similar to those used in line 13-14 to get the extension part.

Reply:
Ah I see. I went and looked at the .slice method again to get some clarity. I think I got the hang of it now. I've updated my code as well/